CMakeLists.txt

```diff
@@ -8,7 +8,11 @@ project(smack)
 if(NOT WIN32 OR MSYS OR CYGWIN)
 
   file(STRINGS "bin/versions" LLVM_VERSION_STR
-    REGEX "LLVM_SHORT_VERSION=\"[0-9]+\"")
+    REGEX "LLVM_FULL_VERSION=\"[0-9]+\.[0-9]+\.[0-9]+")
```

Comment:
LLVM_FULL_VERSION is not being used anywhere in this file. Is that expected?

Reply:
Yes. Part of this variable is in `LLVM_EXTENDED_VERSION`.